cookbooks/aws-parallelcluster-platform/spec/unit/resources/fetch_config_spec.rb

```diff
@@ -2,17 +2,21 @@
 
 describe 'fetch_config:run' do
   context "when running a HeadNode from kitchen" do
```

Comment:
why we have a test "from kitchen"? I think we can avoid to test this case and test instead the case "not kitchen" that is the one important for real use case.

Reply:
The utility of having a spec covering the ktichen case is because all kitchen test may be impacted if a bug is introduced in the fetch_config resource.

Furthermore, if you look at the resource fetch_config, the kitchen case is a super-set of the non-kitchen case, so what we want to cover is already included.